app/src/views/AccountMyFormsEap/EapTableActions/index.tsx

```diff
@@ -41,14 +41,14 @@ function EapTableActions(props: Props) {
             <Link
                 to="eapFullForm"
                 urlParams={{ eapId }}
-                variant="secondary"
+                styleVariant="filled"
             >
                 {strings.eapFormLink}
             </Link>
             <Link
                 to="simplifiedEapForm"
                 urlParams={{ eapId }}
-                variant="secondary"
+                styleVariant="filled"
```

Comment:
`variant="secondary"` is equivalent to:

```
colorVariant="primary"
styleVariant="outline"
```

in case of `Link` component

```suggestion
                styleVariant="outline"
                colorVariant="primary"
```